classes/Product.php

```diff
@@ -501,6 +499,8 @@ class ProductCore extends ObjectModel
             ],
 
             /* Shop fields */
+            'id_supplier' => ['type' => self::TYPE_INT, 'shop' => true, 'validate' => 'isUnsignedId'],
```

Comment:
```suggestion
            'id_supplier' => ['type' => self::TYPE_INT, 'shop' => true, 'validate' => 'isUnsignedId'],
```

Looks like a BC Break. Is it for v9 or 8.1?

Reply:
And, honestly, not sure if it's required, it's an edge-case scenario where you have different suppliers in a different store.

Reply:
The problem is that the default supplier's price and product's wholesale price are synced now, and since wholesale price is a multishop value it's the only way we can handle a logical synchronisation in multishop

So I don't know if it's a BC break but I know I don't have other solution for this 🤷‍♂️

Reply:
Based on the ADR https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md#database-structure adding a required field IS a BC break

So what do we do? We stop working on suppliers multishop and deliver the page unchanged for 8.1 even if we know it's bugged?